docs/pages/interfaces.rst

```diff
@@ -38,52 +38,58 @@ for example, can have one, two or three possible types. See the example below:
 Mappable
 --------
 
-Something is considered mappable if we can `map` it using a function, generally
-`map` is a method that accepts a function. An example in this library is
-:class:`Maybe <returns.maybe.Maybe>`, it has the `map` method.
+Something is considered mappable if we can ``map`` it using a function,
+generally ``map`` is a method that accepts a function.
 
-:class:`Mappable <returns.interfaces.mappable.MappableN>`
-interface help us to create our own mappable container like
-:class:`Maybe <returns.maybe.Maybe>`.
+An example in this library is :class:`Maybe <returns.maybe.Maybe>`,
+that implements the ``Mappable`` interface:
 
 .. code:: python
 
-  >>> from typing import Any, Callable, TypeVar
+  >>> from returns.maybe import Maybe, Some
+
+  >>> def mappable(string: str) -> str:
+  ...     return string + '!'
+
+  >>> maybe_str: Maybe[str] = Maybe.from_value('example')
+  >>> assert maybe_str.map(mappable) == Some('example!')
+
+:class:`Mappable <returns.interfaces.mappable.MappableN>` interface help us to
+create our own mappable container like :class:`Maybe <returns.maybe.Maybe>`.
+
+.. code:: python
+
+  >>> from typing import Any, Callable, TypeVar, Union
 
   >>> from returns.interfaces.mappable import Mappable1
 
-  >>> _NumberType = TypeVar('_NumberType', covariant=True)
-  >>> _NewNumberType = TypeVar('_NewNumberType')
+  >>> _NumberType = TypeVar('_NumberType', bound=Union[int, float, complex])
```

Comment:
Related https://github.com/dry-python/returns/issues/474

Reply:
I've removed the `bound` parameter!